core/src/actions/properties.ts

```diff
@@ -381,7 +382,19 @@ export class PropertyRecordPreview extends AuthenticatedAction {
       updatedAt: null,
     };
 
-    return { errorMessage, record: apiData };
+    const groups = await record.$get("groups");
+    const destinations = await DestinationOps.relevantFor(record, groups);
+
+    await ConfigWriter.run();
```

Comment:
We should not write config files when previewing a record

Reply:
Whoops, fix coming up. I pulled that out of records:create by mistake.